### PR TITLE
add implementation of Ethereum deposit

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
implement deposit eth

Signed-off-by: Alexey Chernyshov <alexey.n.chernyshov@gmail.com>

### CODE REVIEW

File: src/main/kotlin/main/main.kt [hunk 1]
```diff
@@ -18,9 +18,9 @@ fun main(args: Array<String>) {
     val notary = NotaryInitialization()
 
     IrohaInitializtion.loadIrohaLibrary()
-            .flatMap { notary.init() }
-            .failure {
-                logger.logger.error { it }
-                System.exit(1)
-            }
+        .flatMap { notary.init() }
```
Comment (x3medima17) [inline src/main/kotlin/main/main.kt]: How do you format the code?

File: src/main/kotlin/sideChain/iroha/consumer/IrohaConverterImpl.kt [hunk 2]
```diff
@@ -19,50 +20,57 @@ class IrohaConverterImpl {
...
+                            )
+                    }
                 }
+                txs.add(txBuilder.build())
             }
-            txs.add(txBuilder.build())
+        } catch (e: Exception) {
+            logger.error { e }
         }
         return txs
```
Comment (x3medima17) [inline src/main/kotlin/sideChain/iroha/consumer/IrohaConverterImpl.kt]: I think it is better to return `optional` here.

File: src/main/kotlin/main/main.kt (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply src/main/kotlin/main/main.kt]: option + cmd + L

Comment (Alexey-N-Chernyshov) [reply src/main/kotlin/main/main.kt]: IntelliJ Idea settings are described here (https://kotlinlang.org/docs/reference/coding-conventions.html#applying-the-style-guide)

File: src/main/kotlin/sideChain/iroha/consumer/IrohaConverterImpl.kt (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply src/main/kotlin/sideChain/iroha/consumer/IrohaConverterImpl.kt]: Good point, but it seems better to handle errors with RxJava2. I'll fix it.

File: src/main/kotlin/main/main.kt (hunk 1 above)
Comment (x3medima17) [reply src/main/kotlin/main/main.kt]: The point is that I format it the same way, and I get different result :D

**Pull request merged**